### DISCUSSION TITLE
Readme images no longer working.

### DISCUSSION THREAD

**Discussion by gastrader** [community/community#70246] [Repositories]
### Select Topic Area

Bug

### Body

Hi,

I was looking at one of my older repos (https://github.com/gastrader/GPTtherapist) and realized that the readme pics I essentially pasted into the editor from a screenshot are not longer working. I get this error when I click on the image link:
AccessDeniedAccess DeniedPC22RJSWJRSWRVTSzB6rzFCqn3nzsrOJimNQw7nxIVV//t3xyDOGW+OgfBhrkKw/jZQSniKZ9hSdCWYarnYpacj7asM=
This XML file does not appear to have any style information associated with it. The document tree is shown below.

Is there any way to retrieve these images?

**Comment by Janani-Balasooriya** [comment]
Hi @gastrader, Can you attach a screenshot of how you now see the readme pics? Because, I can see the screenshots of your repo's readme properly.

**Comment by Gyanendu01** [comment]
Check Image File URL: Verify that the URL for the image in your README is correct. It should be a direct link to the image file on GitHub, starting with "https://raw.githubusercontent.com/...". Ensure that the URL is not broken or misspelled.

Permissions: Make sure the image file is in a public location or that it has the appropriate permissions set to be accessible. If the image is in a private repository or if the repository's privacy settings have changed, it could lead to access issues.

Check File Path: Double-check the path to the image file within your repository. It should reflect the correct directory structure within your repository, and the filename should be case-sensitive. GitHub URLs are case-sensitive.

I hope this helps you out
